lib/cli/src/commands/create_exe.rs

```diff
@@ -307,9 +306,39 @@ impl CreateExe {
                         .map_err(|err| anyhow::anyhow!(err.to_string()))?;
                     writer.flush()?;
                     drop(writer);
+                    // Write down header file that includes deserialize function
+                    {
+                        let mut writer = BufWriter::new(File::create("static_defs.h")?);
```

Comment:
Please use absolute file paths, not relative file paths to the current directory. I.e. `blah_dir.join("static_defs.h")`

Reply:
It's literally the same thing.

Reply:
I agree with the take from Felix, we should not rely on changing the current working environment for having things to work. We need to stop using `set_current_dir` as only make other functions work as a "side effect". We need to keep things as functional as possible